package.json

```diff
@@ -33,19 +33,20 @@
   },
   "homepage": "https://github.com/sglanzer/ember-cli-blanket",
   "devDependencies": {
-    "broccoli-asset-rev": "^2.0.2",
+    "broccoli-asset-rev": "^2.1.2",
     "chai": "^2.2.0",
-    "ember-cli": "0.2.2",
-    "ember-cli-app-version": "0.3.3",
+    "ember-cli": "1.13.8",
+    "ember-cli-app-version": "^1.0.0",
+    "ember-cli-babel": "^5.1.3",
     "ember-cli-content-security-policy": "0.4.0",
-    "ember-cli-dependency-checker": "0.0.8",
-    "ember-cli-htmlbars": "0.7.4",
-    "ember-cli-ic-ajax": "0.1.1",
-    "ember-cli-inject-live-reload": "^1.3.0",
-    "ember-cli-qunit": "0.3.9",
-    "ember-cli-uglify": "1.0.1",
-    "ember-export-application-global": "^1.0.2",
-    "glob": "^5.0.3",
```

Comment:
I'll spend a bit of time and reduce the dependencies after this PR, we don't use most of these - the project needs a general scrub anyway